### ISSUE TITLE
Need information on how Action VIs work, and when exactly do they trigger

### ISSUE THREAD

**Issue by Brian-Kindinger** [ni/niveristand-custom-device-handbook#39]
Manual looks good but we are missing a section on the Action VIs.  Inparticular we need to know the order of execution and what triggers them to run.

**Comment by phutanu** [comment]
Hey Brian,

Have you seen this section in the [Custom Device Tips and tricks -> Action VIs](https://niveristand-custom-device-handbook.readthedocs.io/en/v1.0.0/Custom_Device_Tips_and_Tricks.html#using-action-vis)?

You can also check in the VeriStand Manual:
- [Action VI Templates](https://www.ni.com/documentation/en/veristand/latest/manual/custom-device-action-vi-template/)
- [Custom Device API Library](https://www.ni.com/documentation/en/veristand/latest/manual/custom-device-api-library/)

Do you still think that more info is required? Thanks!

**Comment by Brian-Kindinger** [comment]
Yes I do think this helps.  It doesn't make sense that it is under a "Tips" section since many of them are required.

**Comment by phutanu** [comment]
I am glad this helped. I will just close this issue, and try to see how we can better advertise this section for users. Thanks!

**Comment by cheloizaguirre** [comment]
@Brian-Kindinger we're going to try to figure out what's a good place to reference the sections Paul sent you so they're more discoverable. Where would you have expected to find that information?
